### ISSUE TITLE
Dashboard: Hidden content/instructions for visual users 

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#30]
URL: https://hcdev.openplus.ca/en/user/203 (Dashboard)
User Agent: Chrome, Firefox, iPhone 12 (iOS 14.7.1)
Language: English

Issue Description: There is hidden content which is important for all users. For example, the instruction to the user that once they submit, the information will appear below. These are only available to screen reader users (Web and mobile). Mobile screens are quite small and user will not be aware of content. Please see screenshots of some of the examples that are hidden. 

Expected results: This content should not be hidden as it seems important for the users. 

![image](https://user-images.githubusercontent.com/14280062/128197895-f3d0947d-cb70-49da-a08f-2d7161162088.png)

**Comment by qanitah** [comment]
Fixed.

**Comment by qanitah** [comment]
This text on French page is in English.
![image](https://user-images.githubusercontent.com/14280062/136220729-02b09e6d-6281-40b8-a9d8-dc6d69e4495b.png)

**Comment by echouin** [comment]
@qanitah Sent EN and FR revised text to OP. Should be completed soon.

**Comment by echouin** [comment]
@qanitah OP has confirmed this has been completed. Please note that Issue #98 appears to be a duplication of this issue. Seeking to get verification to close this item. Thank you.

**Comment by qanitah** [comment]
Fixed
